app/javascript/react/components/MonsterForm.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+
+const MonsterForm = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const [formData, setFormData] = useState({
+    name: "",    description: "",
+    classification: "",
+    habitat: "",
+  })
+
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const handleSubmit = (event) => {
+
+    event.preventDefault()
+    postNewMonster()
```

Comment:
I think line changes modified the original direction for adding extra lines, here it looks like an empty line should be added between the 2 functions `handleChange` and `handleSubmit`. There doesn't need to be an empty line at the beginning of the `handleSubmit` function